eks-anywhere-common/Testers/Sysdig/sysdig-test-cronjob.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: sysdig-agent-test
+  namespace: sysdig
+spec:
+  schedule: "0 1 * * *"
+  backoffLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        activeDeadlineSeconds: 1000
```

Comment:
activeDeadlineSeconds appears to be reporting as an unknown field for this api version - Please check

Reply:
fixed, thanks @mikemcd3912